connectors/v2/pakartot.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = '#player';
+
+Connector.artistSelector = 'div.jp-player-artist > a';
+
+Connector.trackSelector = 'div.jp-player-title > a';
+
+Connector.playButtonSelector = '.jp-play';
+
+Connector.getArtist = function() {
+	var text = $(this.artistSelector).text().trim();
```

Comment:
I'd keep only the very basic cleanup in the base connector, basically just trimming, which is useful everywhere.

I've had an idea long time ago for customizable filters on "core" level. Connectors would do the basic cleanup and send data to core, where a configurable set of filters (probably somewhere on the beginning of pipeline) would be applied. The filters could be further grouped and could be customizable connector-wide or for single specific connector.

I think moving the cleanup to core would make connectors even more simple and we'd avoid copy-pasted Youtube filters in most of them :grinning: